OneSignalSDK/onesignal/core/src/main/java/com/onesignal/session/internal/session/SessionModelStore.kt

```diff
@@ -5,5 +5,19 @@ import com.onesignal.common.modeling.SingletonModelStore
 import com.onesignal.core.internal.preferences.IPreferencesService
 
 open class SessionModelStore(prefs: IPreferencesService) : SingletonModelStore<SessionModel>(
-    SimpleModelStore({ SessionModel() }, "session", prefs),
-)
+    SimpleModelStore(
+        { SessionModel() },
+        "session",
+        prefs,
+    ),
+) {
```

Comment:
To me, this doesn't look as good as a one-liner. But linty lint was nitty picky.